pointsProgForm.php

```diff
@@ -0,0 +1,403 @@
+<?php
```

Comment:
Needs a header to explain file. Great indentation. Great variable names. Still needs comments throughout.